.github/workflows/mac_python_matrix_test.yml

```diff
@@ -35,6 +35,13 @@ jobs:
         echo "Install Pyomo dependencies..."
         echo ""
         pip install cython numpy scipy ipython openpyxl sympy pyyaml pyodbc networkx xlrd pandas matplotlib dill seaborn pymysql pyro4 pint pathos
+        echo ""
+        echo "Install GAMS..."
+        echo ""
+        wget -q https://d37drm4t2jghv5.cloudfront.net/distributions/30.1.0/macosx/osx_x64_64_sfx.exe
+        chmod +x osx_x64_64_sfx.exe
+        ./osx_x64_64_sfx.exe -q -d gams
+        PATH=$PATH:/gams/gams30.1_osx_x64_64
```

Comment:
This is not quite right: we are still getting that the GAMS python bindings and the gams executable are not available; e.g.:
```
Thu, 30 Jan 2020 21:30:15 GMT test_keepfiles_gms (pyomo.solvers.tests.checks.test_GAMS.GAMSTests) ... SKIP: The 'gams' executable is not available
Thu, 30 Jan 2020 21:30:15 GMT test_keepfiles_py (pyomo.solvers.tests.checks.test_GAMS.GAMSTests) ... SKIP: The 'gams' python bindings are not available
```

Reply:
Lovely. I'll look into it.

Reply:
@jsiirola - It looks like licensing is part of the issue (as per [step 5](https://www.gams.com/latest/docs/UG_MAC_INSTALL.html#UG_MAC_SFX_INSTALL)). Do we have an existing demo license, or shall I request one from GAMS?

Reply:
We do not have a demo license.  That is a relatively new development: for the 24.8.5 release, the download defaults to demo mode and does not require a separate license.  It would be great to reach out to GAMS and see what we can work out.  I worry about how we would secure access to the license: while we can use it on Jenkins where we can restrict who has access to the workspace and build environment, on github actions, the world would have access to the license file, which is almost certainly problematic.

Reply:
We'll have to work something out if we want to use a newer distribution of GAMS. Do you mind if I reach out to GAMS about a community license for use in Pyomo testing?

Also, I wonder if there might be a Github Action that can grab input files (like licenses) from a private repo, provided that the workflow executor has the correct permissions. Let me look into that as a solution for the access control concern.

Reply:
@mrmundt: that would be great.

Reply:
Guys, starting from version 30.1, GAMS is providing a free community license for linear problems with at most 5000 variables and constraints and 25000 variables and constraints for the remaining problems (this should cover all the tests within Pyomo). Unfortunately, the default demo mode is no longer default after this new version. Here is the info https://www.gams.com/latest/docs/UG_License.html#GAMS_Community_Licenses

Reply:
I rolled back one distribution to 29.1.0 for Mac. It has the old default (demo license).